lib/dogapi/common.rb

```diff
@@ -141,6 +142,12 @@ def prepare_request(method, url, params, body, send_json, with_app_key)
       unless should_set_api_and_app_keys_in_params?(url)
         req['DD-API-KEY'] = @api_key
         req['DD-APPLICATION-KEY'] = @application_key if with_app_key
+        req['User-Agent'] = 'dogapi-rb/%{version} (ruby %{ruver}; os %{os}; arch %{arch})' % {
+          :version => VERSION,
+          :ruver => RUBY_VERSION,
+          :os => RbConfig::CONFIG['host_os'].downcase,
+          :arch => RbConfig::CONFIG['host_cpu']
+        }
```

Comment:
We should probably send this always.

Reply:
What do you mean? We're adding it to each request right now, so that's always unless I'm missing something.

Reply:
Oh, I see what you mean, will fix.